evals/features/cve/test_suggest_impact.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+
+from pydantic_evals import Case, Dataset
+from pydantic_evals.evaluators import Evaluator, EvaluatorContext
+
+from aegis.agents import rh_feature_agent
+from aegis.features.cve import SuggestImpact, SuggestImpactModel
+
+from evals.features.common import common_feature_evals, handle_eval_report
+
+
+# dict to convert "IMPORTANT" to 8.0 etc
+# the following line is needed for ruff to accept the aligned comments
+# fmt: off
+NUM_BY_IMPACT = {
```

Comment:
u might be able to get this enum from osidb-bindings

Reply:
`osidb-bindings` provides an enum which converts the constants to the identical strings: https://github.com/RedHatProductSecurity/osidb-bindings/blob/master/osidb_bindings/bindings/python_client/models/impact_enum.py

But it will not tell me the distance between `LOW` and `IMPORTANT` as a floating point number in some specific range, which is needed in the evaluation suite.

Reply:
ok no worries, I assumed it might have that ... resolving this